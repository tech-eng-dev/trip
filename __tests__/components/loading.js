import React from 'react';
import renderer from 'react-test-renderer';
import CustomLoadingView from '../../src/components/loading';


test('LoadingView renders correctly', () => {
  const loadingView = renderer.create(<CustomLoadingView
    visible={false}
  />).toJSON();
  expect(loadingView).toMatchSnapshot();
});
