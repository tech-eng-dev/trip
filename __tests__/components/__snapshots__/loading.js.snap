// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadingView renders correctly 1`] = `<View />`;
